driver/normalizer/annotation.go

```diff
@@ -136,15 +136,22 @@ var Annotations = []Mapping{
 	Map(Obj{
 		"IASTClass": String("CPPASTName"),
 		"Name": String(""),
+		"IsQualified": Var("isqual"),
 	}, Obj{
 		uast.KeyType: String("CPPASTName"),
 		uast.KeyRoles: Roles(role.Identifier),
 		uast.KeyToken: String(""),
+		"IsQualified": Var("isqual"),
 	}),
 
-	AnnotateType("CPPASTName", FieldRoles{"Name": {Rename: uast.KeyToken}}, role.Identifier),
-	AnnotateType("CPPASTImplicitName", FieldRoles{"Name": {Rename: uast.KeyToken}},
-		role.Identifier),
+	AnnotateType("CPPASTName", FieldRoles{
+		"Name": {Rename: uast.KeyToken},
+	}, role.Identifier),
+
+	AnnotateType("CPPASTImplicitName", FieldRoles{
+		"Name": {Rename: uast.KeyToken},
+	}, role.Identifier),
+	AnnotateType("ASTInclusionStatement", FieldRoles{"Name": {Rename: uast.KeyToken}}, role.Import),
```

Comment:
```suggestion

	AnnotateType("ASTInclusionStatement", FieldRoles{
		"Name": {Rename: uast.KeyToken},
	}, role.Import),
```

for consistency with the others.